structs/config.go

```diff
@@ -30,10 +30,14 @@ type PageContext struct {
 	MainFacebook    string `toml:"mainFacebook"`
 	NewsTwitter     string `toml:"newsTwitter"`
 	Pages           []Page
+	Sustainer       struct {
```

Comment:
Not sure if `Sustainer` is the best term to use, but it's the most brand-neutral.